aiops/requirements.txt

```diff
@@ -0,0 +1,3 @@
+elasticsearch[async]==7.13.0
```

Comment:
this was to have a minimal set of requirements for code under the new aiops folder. e.g. from a blank environment, what packages to install just to test this code. it will be better to have this here as more code is added.